tests/unit/FilingHistoryList1.spec.ts

```diff
@@ -605,6 +617,66 @@ describe('Filing History List - misc functionality', () => {
     businessStore.setLegalType(CorpTypeCd.COOP)
     expect(vm.disableCorrection({ ...item, name: 'registration' })).toBe(true)
   })
+
+  it('displays multiple filing items, and remove the filing that displayLedger is false', async () => {
+    // init store
+    businessStore.setIdentifier('CP0001191')
+    filingHistoryListStore.setFilings([
+      {
+        availableOnPaperOnly: false,
+        businessIdentifier: 'CP0001191',
+        commentsCount: 0,
+        displayLedger: true,
+        displayName: 'Annual Report',
+        effectiveDate: '2019-11-20 22:17:54 GMT',
+        filingId: 111,
+        isFutureEffective: false,
+        name: FilingTypes.ANNUAL_REPORT,
+        status: FilingStatus.COMPLETED,
+        submittedDate: 'Tue, 02 July 2019 12:00:00 GMT',
+        submitter: 'Submitter 1'
+      } as any,
```

Comment:
Why does this need casting to `any` but the other 2 objects below don't?

Reply:
looks like the other 2 objects inherit the above.